xml/hardening.xml

```diff
@@ -590,14 +496,29 @@
      <term>Minimal Package Installation</term>
      <listitem>
       <para>
-       Generally, an RPM software package consists of the package's meta data
-       that is written to the RPM database upon installation, the package's
-       files and directories and scripts that are being executed before and
-       after installation and removal.
+       Generally, an RPM software package consists of the following:
       </para>
+      <itemizedlist>
+       <listitem>
+        <para>The package's meta data that is written to the RPM database upon
```

Comment:
"meta data" should be "metadata"

Reply:
Hey, you have read the style guide! :) Fixed.